### ISSUE TITLE
nova compute shows incorrect flavor for CCI server

### ISSUE THREAD

**Issue by bodenr** [softlayer/jumpgate#69]
The jumpgate compute driver for nova is returning the incorrect flavor mapping on 'server show'.

For example I have the following CCI which I created through jumpgate (from the instance order info on SL web UI):

```
Name: Cloud Server
Domain: nova-jumpgate-dal.jumpgate.com Quantity: 1
 Hourly Setup
Data Center
Dallas 1
2 x 2.0 GHz Cores
2 GB
```

I created this server via nova / jumpgate with flavor 3 which corresponds to:

```
| 3  | 2 vCPU, 2GB ram, 100GB | 2048      | 100  | 0         |      | 2     | 1           | True      |
```

But when I show it via nova / jumpgate I get:

```
root@jump:~# nova show nova-jumpgate-dal
+-----------------------------+-------------------------------------------------------------------------+
| Property                    | Value                                                                   |
+-----------------------------+-------------------------------------------------------------------------+
| OS-EXT-AZ:availability_zone | 3                                                                       |
| OS-EXT-STS:power_state      | 1                                                                       |
| OS-EXT-STS:task_state       | -                                                                       |
| OS-EXT-STS:vm_state         | ACTIVE                                                                  |
| accessIPv4                  |                                                                         |
| accessIPv6                  |                                                                         |
| created                     | 2014-03-22T12:19:15-06:00                                               |
| flavor                      | 1 vCPU, 1GB ram, 25GB (1)                                               |
| hostId                      | 3979788                                                                 |
| id                          | 3979788                                                                 |
| image                       | frozen.ubuntu.storage.2013/06/25 (7a938011-a7c2-4e4e-9394-4c6456a24d07) |
| image_name                  |                                                                         |
| name                        | nova-jumpgate-dal                                                       |
| private network             | MY_PRIVATE_IP                                                            |
| public network              | MY_PUBLIC_IP                                                           |
| security_groups             | default                                                                 |
| status                      | ACTIVE                                                                  |
| tenant_id                   | 278184                                                                  |
| updated                     | 2014-03-22T12:23:12-06:00                                               |
| user_id                     | 170512                                                                  |
+-----------------------------+-------------------------------------------------------------------------+
```

As you can see above, jumpgate incorrectly shows the instance as 'flavor 1'.

Haven't had time to dig into this one, but appears to be a looming issue.

**Comment by sudorandom** [comment]
The flavor id is currently hard coded. Fixing this is one of the lingering action items defined in issue #27.

**Comment by jimlindeman** [comment]
This has been addressed by Issue #138
